pkg/kubelet/kubelet_test.go

```diff
@@ -311,7 +328,12 @@ func newTestKubeletWithImageList(
 	kubelet.readinessManager = proberesults.NewManager()
 	kubelet.startupManager = proberesults.NewManager()
 
-	fakeContainerManager := cm.NewFakeContainerManager()
+	var fakeContainerManager *cm.FakeContainerManager
+	if nodeConfig == nil {
+		fakeContainerManager = cm.NewFakeContainerManager()
+	} else {
+		fakeContainerManager = cm.NewFakeContainerManagerWithNodeConfig(*nodeConfig)
+	}
```

Comment:
```suggestion
	fakeContainerManager  := cm.NewFakeContainerManagerWithNodeConfig(nodeConfig)
```

nit: make the `nodeConfig` field in `FakeContainerManager` a pointer so you no longer need the if/else

Reply:
The nil pointer check would then move to GetNodeConfig(). I think having it here is better ?

Reply:
oh got it, SGTM